cmd/ignite/cmd/root.go

```diff
@@ -14,39 +14,23 @@ import (
 	"github.com/weaveworks/ignite/cmd/ignite/cmd/vmcmd"
 	"github.com/weaveworks/ignite/pkg/logs"
 	logflag "github.com/weaveworks/ignite/pkg/logs/flag"
-	"github.com/weaveworks/ignite/pkg/util"
+	versioncmd "github.com/weaveworks/ignite/pkg/version/cmd"
 )
 
 var logLevel = logrus.InfoLevel
 
 // NewIgniteCommand returns the root command for ignite
 func NewIgniteCommand(in io.Reader, out, err io.Writer) *cobra.Command {
+	// Set the default logging level
+	logs.Logger.SetLevel(logLevel)
```

Comment:
I assume the flag based log level is set somewhere now that `PersistentPreRun` is removed?